Makefile

```diff
@@ -12,12 +12,12 @@ test sure: test-pypackage test-pyapp test-pyramid-app
 
 .PHONY: test-pypackage
 test-pypackage:
-	@bin/make_test pypackage console_script=yes db=yes devdata=yes pypi=yes services=yes
+	@bin/make_test pypackage console_script=yes devdata=yes postgres=yes pypi=yes
```

Comment:
Oops: the `postgres` option is named `postgres` not `db`, so this test wasn't actually enabling the Postgres/DB feature. Also there is no `services option